src/lib.rs

```diff
@@ -537,10 +543,13 @@ impl Build {
             cmd.arg("-c");
         }
 
-        cmd.arg(&src);
+        cmd.arg(&src)
+            .stdin(Stdio::null())
+            .stdout(Stdio::null())
+            .stderr(Stdio::null());
```

Comment:
Should we inherit stdout and stderr?

Reply:
I believe we should not, since stdout and stderr is for messages from compiler when building the project.
`Build::is_flag_supported` does not build the project but only test the flags, so I think the stdout and stderr should be piped to null.

Reply:
> But we don't provide any other mechanism for the developer to check why a test failed, unlike other build systems that write the output of all these checks/tests to a log file.

Ehh that's true, I can set that to inherit but I think the stdout/stderr might be a little bit confusing to users.
If that's fine, then I can remove these lines.

Reply:
Let’s wait for another opinion? That was just my two cents.